benchmarks/index.js

```diff
@@ -79,13 +86,15 @@ class Runner {
         fsExtra.emptyDirSync(nodeModulesDir);
 
         const dependencies = {};
+        console.log(`benchmark: using versions: ${this.versions}`);
         for (let i = 0; i < this.versions.length; i++) {
             const version = this.versions[i];
             const name = `brighterscript${i + 1}`;
 
             //if the version is "current", then make a local copy of the package from the dist folder to install (because npm link makes things slower)
             if (version === 'local') {
-                dependencies[name] = this.buildCurrentTarball();
+                const localVersion = this.tar ? this.buildCurrentTarball() : this.buildCurrent();
```

Comment:
Add option to benchmark local files directly from the filesystem instead of from a tarball
On my setup, tarballs were cached, so local changes wouldn't be included, and also there did not seem to be anny performance difference running from tarball or local filesystem

Reply:
great. For windows, there was significant performance issues....i'm guessing due to symlinks? But this option is nice to have...it probably speeds up testing too. :) Thanks!